Ghidra/Processors/SuperH4/data/languages/SuperH4.sinc

```diff
@@ -1525,7 +1525,7 @@ define pcodeop cos;
 	
 	pi:4 = 0x40490fdb;				# 3.1415927.. as 32-bit float
 	pi2:4 = int2float(2 :4) f* pi;			# multiply pi by 2 (2pi)
-        angle = (pi2 f* angle) f/ offset;  		# convert to radian
```

Comment:
Improved decompilation:
    fVar1 = ((float)(param_1 & 0xffff) * 6.283185) / 65536.0;
    uVar2 = sin(fVar1);
    uVar3 = cos(fVar1);

Think that's as good as it gets.